tests/testthat/test-util.R

```diff
@@ -95,3 +95,20 @@ test_that("write_lines recovers on error", {
   expect_silent(write_lines(value, filename))
   expect_identical(readLines(filename), value)
 })
+
+
+test_that("read_text_file() works", {
+  expect_false(file.exists("does_not_exist"))
+  expect_equal(read_text_file("does_not_exist"), "")
```

Comment:
This behaviour should be changed to throw an error (see comment on main discussion thread)

Reply:
4f06d03782e2d7bbe2ee7f35b97e0fcb893dac86 is a start.